Modules/faulthandler.c

```diff
@@ -1370,7 +1370,7 @@ void _PyFaulthandler_Fini(void)
 #ifdef HAVE_SIGALTSTACK
     if (stack.ss_sp != NULL) {
         /* Fetch the current alt stack */
-        stack_t current_stack = {};
+        stack_t current_stack = {0};
```

Comment:
Do we know that this is always valid? I'm not sure what we can safely rely with respect to the `stack_t` structure, but if the first field of that struct isn't a scalar, I'd expect initializing it with `0` to be potentially problematic.

Reply:
This code was recently modified to use `memset()` (in b84cb70880a0acfcbbaca7bcda405af08f94d269), so faulthandler.c is no longer part of this PR.